src/g_umapinfo.h

```diff
@@ -18,11 +18,30 @@
 //
 //-----------------------------------------------------------------------------
 
-#ifndef UMAPINFO_H
-#define UMAPINFO_H
+#ifndef G_UMAPINFO_H
+#define G_UMAPINFO_H
 
 #include "doomtype.h"
 
+typedef enum
+{
+    MapInfo_LabelClear = (1u << 0),
+
+    MapInfo_EndGameArt = (1u << 2),
```

Comment:
Well, that's a really cumbersome way to write `4`. 😁

Reply:
I like `(1u << 0), (1u << 1), ... (1u << 32)` for flags more than hex or decimal values.